pkg/installer/installer.go

```diff
@@ -126,15 +126,43 @@ func Purge() {
 	purge(defaultLocksPath, defaultRepositoriesPath)
 }
 
+func PurgePackage(pkg string) {
+	purgePackage(pkg, defaultLocksPath, defaultRepositoriesPath)
+}
+
+func purgePackage(pkg string, locksPath, repositoryPath string) {
+	var err error
+	span, ctx := tracer.StartSpanFromContext(context.Background(), "purge_pkg")
+	defer span.Finish(tracer.WithError(err))
+	span.SetTag("params.pkg", pkg)
+
+	switch pkg {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 Suggestion:** ‏This is super nice! Can you add the injector's removal too? That would help in the system tests

Reply:
👍